### DISCUSSION TITLE
Can I build this with Visual Studio?

### DISCUSSION THREAD

**Discussion by MrBolt2005** [Beep6581/RawTherapee#6025] [Q&A]
I wonder if I could get this program built using CMake-generated Visual Studio projects. If yes, how do I do it?

**Comment by Thanatomanic** [comment]
I have it working, so it can be done. I think they key step was to Install the CMake Tools addon in VSCode, that should pick up the makefiles. Then it does require some fiddling with settings (don't know them exactly anymore), but it should work without too much trouble.

**Reply by MrBolt2005** [reply]
By the way, can I compile this program using Visual C++? Just asking.

**Reply by Thanatomanic** [reply]
I don't know about that. I also don't really see the benefit of doing that over compilation with the GNU toolkit, which is also readily available for Windows.
